lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -3392,6 +3393,9 @@ LogicalResult StmtEmitter::visitStmt(TypedeclOp op) {
     emitError(op, "SV attributes emission is unimplemented for the op");
 
   startStatement();
+  if (isZeroBitType(op.getType()))
+    ps << PP::neverbox << "// ";
```

Comment:
Need to close every box you open, also if I'm not mistaken this will be caught when closing out the PP so maybe a small test is warranted?

Reply:
I added the box close - not sure how to test this though.

Reply:
I updated the 0-width typedef test to exercise an assert if I didn't close the box.